src/OrbitGl/CaptureWindow.cpp

```diff
@@ -359,14 +359,14 @@ std::unique_ptr<AccessibleInterface> CaptureWindow::CreateAccessibleInterface()
   return std::make_unique<AccessibleCaptureWindow>(this);
 }
 
-void CaptureWindow::Draw() {
+void CaptureWindow::Draw(bool viewport_was_updated) {
```

Comment:
I prefer "viewport_was_dirty"

Reply:
I've changed the name and fixed the gcc warning, PTAL.

Reply:
Looks good.